### ISSUE TITLE
Format External links

### ISSUE THREAD

**Issue by smuckola** [zhaofengli/refill#14]
Please add a checkbox for "format 'External links' as {{URL}}". It can convert "*[http://example.com Example site]" into "*{{URL|1=example.com|2=Example site}}".  Include the 1= and 2= in case the URL has a "=" in it.  Thanks.

**Comment by AlexanderMisel** [comment]
我也觉得要是也能format外部链接更好。

**Comment by nemobis** [comment]
I agree it would be nice to do both this and the biliography section.

**Comment by waldyrious** [comment]
I'm not sure that's appropriate. From [Template:URL's documentation](https://en.wikipedia.org/wiki/Template:URL):

> **Note:** If you wish to display text instead of the URL (e.g. [website](http://example.com) instead of [example.com](http://example.com)), please **do not use this template**. Instead, consider using normal [wiki markup](https://en.wikipedia.org/wiki/Help:Link#External_links "Help:Link") or [Template:Official website](https://en.wikipedia.org/wiki/Template:Official_website).
